lib/reconcile/vernemq.go

```diff
@@ -41,6 +41,8 @@ import (
 	"github.com/astarte-platform/astarte-kubernetes-operator/version"
 )
 
+const AstarteVerneMQCertificateName string = "astarte-tls-cert"
```

Comment:
This default name is scary. I'd go for `astarte-vernemq-cert` (tls is redundant as it will be a tls secret)

Reply:
Also remember that you should use the resource prefixes here, so in general it should be `%prefix-vernemq-cert`

Reply:
Removed the constant. Both the secret name and its namespace must be set within the Astarte CR inside a `SecretRef` field.